src/eventreporter/EventData.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import MeetingHistoryState from '../eventcontroller/MeetingHistoryState';
+
+/**
+ * [[EventData]] describes an event information.
+ */
+export default interface EventData {
+  /**
+   * Name of an event with the [[MeetingHistoryState]] type.
+   * For example, "signalingDropped", "meetingReconnected", "meetingStartRequested".
+   */
+  name: MeetingHistoryState;
+  /**
+   * Event generated timestamp in milliseconds.
+   */
+  ts: number;
```

Comment:
nit: Is there a reason why we do not just call it timestamp?

Reply:
Yeah kept it in backend accepted format. They have `ts` as the timestamp attribute.